pydash/pydash_web/controller/dashboards.py

```diff
@@ -44,29 +44,7 @@ def dashboards():
     """
     dbs = pydash_app.dashboard.dashboards_of_user(current_user.id)
 
-    mock_data = [
-                  {
-                    "id": "4242424242424242",
-                    "url": "http://pydash.io/",
-                    "endpoints": [
-                      {
-                        "name": "my.endpoint.name",
-                        "enabled": True
-                      }
-                    ]
-                  },
-                  {
-                    "id": "5353535353535353",
-                    "url": "http://pistach.io/",
-                    "endpoints": [
-                      {
-                        "name": "my.other.endpoint.name",
-                        "enabled": False
-                      }
-                    ]
-                  }
-                ]
-    return jsonify(mock_data), 200
+    return jsonify([_dashboard_detail(db) for db in dbs]), 200
```

Comment:
The overview `/api/dashboards`-call should not return all dashboard detail information, but only general information:

- dashboard_id
- dashboard_url
- list of endpoints. (These might be removed at some point, but is what we currently specified in the API spec.)

Reply:
Your comment is entirely true. However, since the API spec is somewhat outdated already and I assumed the `/api/dashboards`-call simply returns a list of dashboard-details (essentially the information the `/api/dashboards/<dashboard-ID>`-call returns), I extended the yet undocumented change of the `/api/dashboards/<dashboard-ID>`-call's output to `/api/dashboards`. Looking at the (outdated) API spec, this difference in amount of detail seems evident to me now, but it was not all that clearly written.

I do see the merit in only returning a minimum of information with each `/api/dashboards`-call and will implement that in my fix. However, I would like to get the entire team on one line w.r.t. the backend-API spec and formally update it in tomorrow's meeting.